### ISSUE TITLE
Typings issue when using the `children` of the entry to eager load

### ISSUE THREAD

**Issue by cookie10codes** [nystudio107/craft3-eagerbeaver#32]
### Describe the bug
I'm currently updating one of our largest projects to C4 and we're using eager beaver on that one. seems like I'm having issues  when using `entry.children()`

### To reproduce
```
{%- set modules = entry.children() -%}

{%- do craft.eagerBeaver.eagerLoadElements(modules, [
        'contentBuilder'
    ])
-%}
```

This results in this error:
```
nystudio107\eagerbeaver\variables\EagerBeaverVariable::eagerLoadElements(): Argument #1 ($elements) must be of type craft\base\ElementInterface|array, craft\elements\db\EntryQuery given, called in /var/www/project/cms/vendor/twig/twig/src/Extension/CoreExtension.php on line 1607
```

I also eager load the entry.children bit as shown here:
```
{%- do craft.eagerBeaver.eagerLoadElements(entry, [
        'children',
    ])
-%}

{%- set modules = entry.children() -%}

{%- do craft.eagerBeaver.eagerLoadElements(modules, [
        'contentBuilder'
    ])
-%}
```

This results in this error:
```
nystudio107\eagerbeaver\variables\EagerBeaverVariable::eagerLoadElements(): Argument #1 ($elements) must be of type craft\base\ElementInterface|array, Illuminate\Support\Collection given, called in /var/www/project/cms/vendor/twig/twig/src/Extension/CoreExtension.php on line 1607
```

### Versions
- Plugin version: 4.0.1
- Craft version: Craft Pro 4.2.1.1

**Comment by khalwat** [comment]
Addressed in https://github.com/nystudio107/craft-eagerbeaver/commit/0e08e023329f9d49b54d1bb695cf23d7cc4705ba

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-eagerbeaver": "dev-develop-v4 as 4.0.2”,
```
Then do a `composer clear-cache && composer update`

**Comment by khalwat** [comment]
Released as 4.0.2: https://github.com/nystudio107/craft-eagerbeaver/releases/tag/4.0.2
